source/SpriteShader.cpp

```diff
@@ -74,8 +74,9 @@ namespace {
 		{GL_GREEN, GL_BLUE, GL_GREEN, GL_ALPHA}, // 24 darkened magenta
 		{GL_RED, GL_BLUE, GL_RED, GL_ALPHA}, // 25 pure magenta
 		{GL_RED, GL_GREEN, GL_RED, GL_ALPHA}, // 26 faded magenta
-		{GL_BLUE, GL_ZERO, GL_ZERO, GL_ALPHA}, // 27 red only (cloaked)
```

Comment:
The red only swizzle is still useful, no need to remove it imo. (also backwards compatibility)